spec/support/features/session_helper.rb

```diff
@@ -50,6 +50,7 @@ def sign_in_user(user = create(:user))
     end
 
     def sign_in_before_2fa(user = create(:user))
+      allow(FeatureManagement).to receive(:prefill_otp_codes?).and_return(true)
```

Comment:
since we use this method in 11 files, do you think it makes sense to include this setup step in just the places we need it? (lines 47 an 81)?